scripts/phased_training_resume.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+import argparse
```

Comment:
How is `argparse` getting installed on the runner?

Reply:
It's part of the standard python library.  Maybe I am not following the question correctly though?

Reply:
Did not know this was part of the python standard lib! No worries then